### PR TITLE
changing username to wikia standard

### PR DESCRIPTION
**Pull request by justnpT**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/8eb0a5a8e79528aafc82071bba0794118e515cac/src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java#L7) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java [hunk 1]
```diff
@@ -82,11 +82,10 @@ public void loggedInUserOnMobileCanVoteForFirstReply() {
     firstReplyUpvoteButtonClickAddsAnUpvoteAndSecondClickRemovesTheUpvote();
   }
 
-  @Test(groups = "discussions-loggedInUserOnMobileCanUpvote", enabled = false)
-  @RelatedIssue(issueID = "SOC-2596")
+  @Test(groups = "discussions-loggedInUserOnMobileCanUpvote")
   @Execute(asUser = User.USER_3)
   @InBrowser(browserSize = MOBILE_RESOLUTION)
-  public void loggedInUserOnMobileCanUpvote() {
+  public void regularUserOnMobileCanUpvote() {
```
Comment (qaga) [inline src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java]: Why did you change loggedInUser to regularUser? What is regular user?

File: src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java [hunk 2]
```diff
@@ -101,11 +100,10 @@ public void loggedInUserOnDesktopCanVoteForFirstReply() {
     firstReplyUpvoteButtonClickAddsAnUpvoteAndSecondClickRemovesTheUpvote();
   }
 
-  @Test(groups = "discussions-loggedInUserOnDesktopCanUpvote", enabled = false)
-  @RelatedIssue(issueID = "SOC-2596")
+  @Test(groups = "discussions-loggedInUserOnDesktopCanUpvote")
   @Execute(asUser = User.USER_3)
-  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
```
Comment (qaga) [inline src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java]: if you remove browser = Browser.FIREFOX, test will be run on mobile skin by Jenkins. See [here](https://wikia-inc.atlassian.net/wiki/pages/viewpage.action?pageId=71303284) why it happens like that

File: src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java [hunk 3]
```diff
@@ -101,11 +100,10 @@ public void loggedInUserOnDesktopCanVoteForFirstReply() {
...
-  public void loggedInUserOnDesktopCanUpvote() {
+  @InBrowser(browserSize = DESKTOP_RESOLUTION)
+  public void regularUserOnDesktopCanUpvote() {
```
Comment (qaga) [inline src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java]: same question about regularUser

File: src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java (hunk 1 above)
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java]: I'd like us to move our custom vocabulary (how we call users) to wikia user access level vocabulary:
http://mediawiki119.wikia.com/wiki/Help:User_access_levels
This vocabulary is already used in our Test User pages
https://wikia-inc.atlassian.net/wiki/display/SOC/Manual+testing

The goal is to be more precise. Logged in user doesn't tell much about who the user is. Is this staff ? Is this a helper ? In this case this is the regular user (no additional rights granted)

File: src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java (hunk 3 above)
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java]: It's to keep names consistend with user access levels: http://mediawiki119.wikia.com/wiki/Help:User_access_levels

That's also the standard for test user docs: https://wikia-inc.atlassian.net/wiki/display/SOC/Manual+testing

**Comment by qaga** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/5845200506e164d1d8f8374b13004648b91ce7e6/src/test/java/com/wikia/webdriver/testcases/discussions/UpvotingTests.java#L7) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
